crates/host/src/oci.rs

```diff
@@ -2,11 +2,12 @@
 // https://github.com/wasmCloud/wasmcloud-otp/blob/5f13500646d9e077afa1fca67a3fe9c8df5f3381/host_core/native/hostcore_wasmcloud_native/src/oci.rs
 
 use crate::par;
+use crate::wasmbus::config::OciOptions;
```

Comment:
Shouldn't there be a `Config` struct defined in this module instead? Why is it in `wasmbus::config`?

Reply:
Moved to `Config` struct in this module and refactored around this.